recipe/meta.yaml

```
@@ -24,12 +24,11 @@ requirements:
     - wheel
   run:
     - python
-    - matplotlib-base >=3.6.0
-    - mizani >=0.9.0
+    - matplotlib-base >=3.7.0
+    - pandas >=2.1.0,<3.0a0
+    - mizani >=0.11.0,<0.12.0
     - numpy >=1.23.0
-    - pandas >=1.5.0
-    - patsy >=0.5.1
-    - scipy >=1.5.0
+    - scipy >=1.7.0
     - statsmodels >=0.14.0
```

```suggestion
    - statsmodels >=0.14.0
  run_constrained:
    - scikit-misc >=0.3.0
```
https://inspector.pypi.io/project/plotnine/0.13.6/packages/93/b1/e1c84e68cf24e83c4d251122e04c48840be203dafb4727e738381cdefa74/plotnine-0.13.6-py3-none-any.whl/plotnine-0.13.6.dist-info/METADATA#line.78